src/placeos-driver/protocol/management.cr

```diff
@@ -448,6 +451,16 @@ class PlaceOS::Driver::Protocol::Management
       mod_id = request.id
       setting_name, setting_value = Tuple(String, YAML::Any).from_yaml(request.payload.not_nil!)
       on_setting.call(mod_id, setting_name, setting_value)
+    when "status"
```

Comment:
maybe we should use a string backed enum for this?

Reply:
Yeah will update this